tests/test_markers.py

```diff
@@ -110,6 +109,7 @@ def test_matches_expected(self):
             )
 
         assert environment == {
+            "extra": "",
```

Comment:
Since `default_environment` is part of the public API (pending documentation https://github.com/pypa/packaging/issues/503), I wonder if we should avoid adding `extra` to it.